esp/services/ws_workunits/ws_workunitsHelpers.cpp

```diff
@@ -89,13 +90,19 @@ SecAccessFlags getWsWorkunitAccess(IEspContext& cxt, IConstWorkUnit& cw)
 void ensureWsWorkunitAccessByOwnerId(IEspContext& cxt, const char* owner, SecAccessFlags minAccess)
 {
     if (!cxt.validateFeatureAccess(getWuAccessType(owner, cxt.queryUserId()), minAccess, false))
+    {
+        cxt.setAuthStatus(AUTH_STATUS_NOACCESS);
```

Comment:
Not related to this review, but cxt is usually ctx

Reply:
I am going to fix them all.